db/schema.rb

```diff
@@ -11,26 +11,26 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20200207213514) do
+ActiveRecord::Schema.define(version: 20200720140244) do
 
   create_table "branches", force: :cascade do |t|
     t.datetime "git_updated_at",              null: false
-    t.text     "name",           limit: 1024, null: false
+    t.text     "name",           limit: 3072, null: false
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.integer  "author_id"
-    t.integer  "repository_id"
+    t.integer  "author_id",                   null: false
+    t.integer  "repository_id",               null: false
   end
 
   add_index "branches", ["author_id"], name: "index_branches_on_author_id"
   add_index "branches", ["repository_id"], name: "index_branches_on_repository_id"
 
   create_table "commits", force: :cascade do |t|
-    t.text     "sha",           limit: 40,   null: false
-    t.text     "message",       limit: 1024, null: false
+    t.text     "sha",           limit: 120,  null: false
+    t.text     "message",       limit: 3072, null: false
```

Comment:
Kind of odd to have `text` fields that are so short? They could be `string`s. Also, BTW, I believe `text` are always `null: true` regardless of what you way.

Reply:
Yea, these fields are all coming from the `git_model` gem, so we should update those declarations there as a different ticket.

Reply:
Ok, you want to create it should I?

Reply:
I can go ahead and create it, I have the context.

Reply:
Thanks!

Reply:
New ticket for this https://ringrevenue.atlassian.net/browse/TECH-4680